npg-api/npg-api.yaml

```diff
@@ -0,0 +1,361 @@
+openapi: 3.0.3
+info:
+  title: NPG API
+  version: "0.1"
+  description: |-
+    Nexi NPG
+externalDocs:
+  description: API di Nexi per la gestione dei pagamenti HPP (Hosted Payment Page)
+  url: https://developer.nexigroup.com/it/api/specifiche-di-utilizzo-del-servizio
+paths:
```

Comment:
the correct api to call is https://developer.nexigroup.com/it/api/post-orders-hpp